result-async/src/test/scala/uscala/concurrent/result/AsyncResultSpec.scala

```diff
@@ -265,10 +265,10 @@ class AsyncResultSpec(implicit ee: ExecutionEnv) extends Specification with Scal
     }
     "sequence" >> {
       "should transform a Seq(Fail) into a Fail(Seq)" >> prop { xs: Seq[Int] => xs.nonEmpty ==>
-        (xs.map(asyncFail).sequence.underlying must beFail(xs.head).await)
+        (xs.map(asyncFail[Int, Int]).sequence.underlying must beFail(xs.head).await)
```

Comment:
`BuildFrom` can't make collections of `Nothing` anymore - so we add type ascriptions everywhere needed to force the compiler to have a type on the missing side.